src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -663,6 +663,16 @@ export default (accessToken, userID, opts) => {
       {},
       { headers: true }
     ),
+    // TODO: rename
+    singlePartnerSearchCriteriaLoader: gravityLoader<
```

Comment:
I wonder if the Gravity implementation might add this support on to the existing endpoint (returning `hits: [single-instance]`) and so would then re-use the existing loader.

Reply:
Good call and question!

Thinking, I do like the rest-full-ness of a separate singular endpoint but up for debate

Reply:
Thinking too that moving this to gravity might better support future roll ups for all matching alerts (for a given artwork), a la demand 1 and the issue we're seeing with the new tab

Reply:
Leaning in that direction as well, was just thinking we want to be able to handle more "fancy" pagination if a user has a lot of collectors we need to paginate through and maybe thats better to handle at the alert level!

Ex:
<img width="394" alt="Screenshot 2024-08-22 at 2 04 22 PM" src="https://github.com/user-attachments/assets/9d277917-f1ae-47ad-8973-88e2dac22ed6">